helmfile/values/rclone-sync.yaml.gotmpl

```diff
@@ -11,10 +16,29 @@ config:
   dryrun: {{ .Values.objectStorage.sync.dryrun }}
 {{- end }}
 
+{{ $sourceS3 := "" }}
+{{ $sourceSwift := "" }}
+{{ $destinationS3 := "" }}
+{{ $destinationSwift := "" }}
+
+{{- range .Values.objectStorage.sync.buckets }}
+{{- if or (eq (get "sourceType" "" .) "s3") (and (ne (get "sourceType" "" .) "swift") (eq $.Values.objectStorage.type "s3") ) }}
+{{ $sourceS3 = "true" }}
+{{- else if or (eq (get "sourceType" "" .) "swift") (and (ne (get "sourceType" "" .) "s3") (eq $.Values.objectStorage.type "swift") ) }}
+{{ $sourceSwift = "true" }}
+{{- end }}
+
+{{- if or (eq (get "destinationType" "" .) "s3") (and (ne (get "destinationType" "" .) "swift") (eq $.Values.objectStorage.sync.type "s3") ) }}
+{{ $destinationS3 = "true" }}
+{{- else if or (eq (get "destinationType" "" .) "swift") (and (ne (get "destinationType" "" .) "s3") (eq $.Values.objectStorage.sync.type "swift") ) }}
+{{ $destinationSwift = "true" }}
+{{- end }}
+{{- end }}
+
   source:
-    name: default
-    type: {{ .Values.objectStorage.type }}
-  {{- if eq .Values.objectStorage.type "s3" }}
+  {{- if ($sourceS3) }}
+  - name: default-s3
```

Comment:
What do you think on using this terminology for source and destrination:
- source for `src-..` and 
- destination for `dest-..`,
instead of `default-..` and `backup-..`

Reply:
Sounds good to me at least just kept it like that because the original was default and backup.

Reply:
fixed the stuff and changed the names applicationCredentialId to applicationCredentialID so it can use the same id and secret as thanos if that one is enabled.